src/scripts/cli.ts

```diff
@@ -3,13 +3,15 @@ import { run, subcommands } from "cmd-ts";
 
 import * as datasets from "./commands/datasets";
 import * as programs from "./commands/programs";
+import * as database from "./commands/database";
 
 export function runCli() {
     const cliSubcommands = subcommands({
         name: path.basename(__filename),
         cmds: {
             datasets: datasets.getCommand(),
             programs: programs.getCommand(),
+            database: database.getCommand(),
```

Comment:
Can we use the model instead? `orgunits: `.

Reply:
Sure. This is what I had in mind though:

The way I imagined it, the file ("model") was related to `// Commands related to dhis2 database direct manipulation.` (From the old first line in database.ts) So I imagined it would extend in the future towards more subcommands related to database manipulation via SQL strings, not necessarily related to orgunits.

But I obviously trust your intuition more on this matters, so I'll change it.